cpp/benchmarks/utilities.cc

```diff
@@ -178,5 +178,6 @@ std::tuple<utilities::vis_params, t_real> dirty_measurements(
   auto const sigma = comm.broadcast<t_real>();
   return std::make_tuple(utilities::scatter_visibilities(comm), sigma);
 }
+void update_comm(sopt::mpi::Communicator &comm) { comm = sopt::mpi::Communicator::World(); }
```

Comment:
That is the culprit ?

Reply:
Shouldn't it be placed in the fixture ?

Reply:
Yes, so this function is run in the fixture. This way it is done so that it is run after MPI is initilized. Otherwise Google Benchmark does it before.